platform/src/core/apis/Account.js

```diff
@@ -28,8 +30,16 @@ export const postUserLoginDetails = async (data) => {
 };
 
 export const getUserDetails = async (userID, token) => {
-  axios.defaults.headers.common.Authorization = token;
-  return await axios.get(`${USERS_URL}/${userID}`).then((response) => response.data);
+  return await axios
+    .get(`${USERS_URL}/${userID}`, {
+      headers: {
+        Authorization: token,
+      },
+      params: {
```

Comment:
Hi @OchiengPaul442 ....I was already hoping that the solution is not this.
We need to restrict the sage of the token on ONLY the events endpoint.
Otherwise, we will fail to implement proper access control in our system.

In summary:

1. Is there any other solution that does not involve the addition of the token query parameter?
2. This PR's solution does not explain why things are working in STAGING but failing in PRODUCTION.

cc: @Psalmz777 / @Codebmk